frontend/src/components/gotEnzymes/EnzymesTable.vue

```diff
@@ -52,9 +75,10 @@ export default {
   },
   data() {
     return {
+      linkableFields: ['compound', 'domain', 'ec_number', 'protein', 'organism', 'reaction_id'],
       columns: [
         {
-          label: 'Protein',
+          label: 'Gene',
           field: 'protein',
```

Comment:
Shall we name all `protein` as `gene`  for consistency?

Reply:
Yes please

Reply:
The `field` needs to be `protein` here because it is matching what we have in the database. It would definitely be possible to do change that too but is a bigger effort. I suggest that we discuss more about this and perhaps handle it in a new issue.

Reply:
@nanjiangshu is rightly following the acceptance criterion [mentioned above](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1043#issuecomment-1167103575).

Reply:
I'm aware of that change, although it seems to be specifically about the frontend (_This applies to both the table header and the individual page._). The actual field name is listed as `protein` in the `README.txt` file from the data files.

If it must be changed in the backend and db, for practical reasons I would prefer if we can handle that in the work for #1027 to help avoid merge conflicts. It feels cleaner because this PR has not touched the db design.